scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -0,0 +1,157 @@
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.realistic_lighting_and_shadows;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+
+import com.esri.arcgisruntime.layers.ArcGISSceneLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.AtmosphereEffect;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.LightingMode;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+
+public class RealisticLightingAndShadowsController {
+
+  @FXML private SceneView sceneView;
+  @FXML private Label timeLabel;
+  @FXML private Slider timeSlider;
+  @FXML private ComboBox<LightingMode> comboBox;
+  private Surface surface;
+  private Calendar calendar;
+  private SimpleDateFormat dateFormat;
+
+  public void initialize() {
+    try {
+
+      // create a scene and add a basemap to it
+      ArcGISScene scene = new ArcGISScene();
+      scene.setBasemap(Basemap.createTopographic());
+
+      // add the scene to the scene view
+      sceneView.setArcGISScene(scene);
+
+      // add a base surface for elevation data to the scene
+      surface = new Surface();
+      ArcGISTiledElevationSource elevationSource = new ArcGISTiledElevationSource("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer");
+      surface.getElevationSources().add(elevationSource);
+      scene.setBaseSurface(surface);
+
+      // add a scene layer
+      ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer("http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/DevA_BuildingShells/SceneServer/layers/0");
+      scene.getOperationalLayers().add(sceneLayer);
+
+      // add a camera and initial camera position
+      Camera camera = new Camera(45.54605153789073, -122.69033380511073, 941.0002111233771, 162.58544227544266, 60.0,0.0);
+      sceneView.setViewpointCamera(camera);
+
+      // set atmosphere effect to realistic
+      sceneView.setAtmosphereEffect(AtmosphereEffect.REALISTIC);
+
+      // set a calendar with a date and time
+      calendar = new GregorianCalendar(2018, 7, 10, 12, 00, 0);
```

Comment:
I got two IDE tips appearing for this constructor. `minute` can be replaced with "0" and the integer reflecting the month should be replaced with an enum e.g.
```suggestion
      calendar = new GregorianCalendar(2018, Calendar.AUGUST, 10, 12, 0, 0);
```